backend/fleet_management/models.py

```diff
@@ -2,20 +2,43 @@
 import time
 
 from django.db import models
-from django.contrib.auth.models import AbstractUser
+from django.conf import settings
 from django.utils.timezone import now
+from django.contrib.auth.models import AbstractUser
 from django_countries.fields import CountryField
-from django.core.validators import MinLengthValidator
+
+from fleet_management.crypto import PublicKey, PrivateKey, find_pair_of_keys
+
 
 def get_current_timestamp_in_gmt():
     return calendar.timegm(time.gmtime())
 
 
+def pad(n: int):
```

Comment:
Don't really see a lot of reason to have this here as we use `max_length=6` in model's fields so we should to pad to 6.